src/ray/core_worker/core_worker.cc

```diff
@@ -1285,6 +1290,31 @@ void CoreWorker::HandlePushTask(const rpc::PushTaskRequest &request,
     // We have posted an exit task onto the main event loop,
     // so shouldn't bother executing any further work.
     if (exiting_) return;
+
+    if (gcs_actor_management_enabled_ &&
+        request.task_spec().type() == TaskType::ACTOR_CREATION_TASK) {
+      // When restart, the gcs server will lease a worker from raylet and send actor
+      // creating task to the leased worker, if the leased worker is still alive, it
+      // should reply immediately and create nothing when received the actor creating
+      // task. For more details please see the protocol of actor management based on gcs.
+      // https://docs.google.com/document/d/1EAWide-jy05akJp6OMtDn58XOK7bUyruWMia4E-fV28/edit?usp=sharing
+      ActorID current_actor_id;
```

Comment:
Consider that if some of actors are created already in cluster 
 and went well. Suddenly, the GCS Server is crashed. The following steps will be:
1. GCS Server will restart and load all actor_table_data from storage and try reconstruct all of actors.
2. GCS Server will try to lease worker from the node the actor  previously tied to.
3. GCS Server will try to create the actor on the leased worker.

If the worker is already bound with an actor, then the worker should ignore the received ActorCreationTask come from GCS Server just because that the actor is already created on this worker before the GCS Server restart.

But you reminded me that it is possible that the Actor on the Worker is executing a time-consuming task, so `task_execution_service_` will be difficult to execute the ActorCreationTask come from the GCS Server, which will cause the GCS Server 
 late to receive the response from worker.
So, this logic should be moved out of the lambda of `task_execution_service_.post`.